### DISCUSSION TITLE
November 2024 Conflux Documentation Contributor Incentives

### DISCUSSION THREAD

**Discussion by darwintree** [Conflux-Chain/conflux-documentation#808] [Announcements]
We are excited to announce the eligibility list of contributors who are qualified for contributor incentives for 202411. We deeply appreciate your support and highly value the insights and expertise you bring to our community.

以下为202411有资格获取激励的贡献者名单。非常感谢名单中的各位对本项目的支持。

The following GitHub contributors are eligible:

以下的GitHub贡献者有资格领取激励：

* [jackleeio](https://github.com/jackleeio)
* [Vikash-8090-Yadav](https://github.com/Vikash-8090-Yadav)

以下的Crowdin贡献者有资格领取激励：

* [PingJing](https://crowdin.com/profile/PingJing)

The eligibility list is curated in accordance with our [CONTRIBUTING guideline](https://doc.confluxnetwork.org/docs/general/CONTRIBUTING). If you believe your contribution has been overlooked, please do not hesitate to contact us.

名单是根据[CONTRIBUTING指导原则](https://doc.confluxnetwork.org/docs/general/CONTRIBUTING)所挑选的。如果您有对应贡献但被遗漏在了名单外，欢迎联系我们。

## Contributor Incentives Details 贡献者激励详情

At present, we offer corresponding incentives for Chinese translators. Translator incentives are calculated based on the volume of translations and whether the translation is accepted. We will airdrop rewards to translators who have provided their addresses. For the method of providing addresses, please refer to [Conflux Technical Documentation Translation Plan (2024.03.06) ](https://forum.conflux.fun/t/conflux-2024-03-06/20286).

目前我们为中文翻译者提供相应激励，翻译者激励计算与翻译量与翻译是否被接受相关。我们将为已提供了地址的翻译者空投激励。地址提供方式请参考[Conflux 技术文档翻译计划（2024.03.06）](https://forum.conflux.fun/t/conflux-2024-03-06/20286)。

We have introduced financial incentives for contributors which are directly related to the quality of the contributor's work.

我们为仓库的贡献者引入的经济激励是与贡献者的工作质量所直接相关的。

For incentives for documentation content contributors, please refer to:

https://confluxglobal.notion.site/6665e6288ea942d5aa415c408bcfb145?v=14f867639969811c9551000c92971f4d&pvs=4

and their incentives are calculated according to the [2024 reward guidelines](https://github.com/Conflux-Chain/conflux-documentation/blob/2024Q1-reward/docs/general/CONTRIBUTING/reward-guidelines.md). If you have relevant contributions but have been omitted from the list, please feel free to contact us. In addition, we will collect the (temporary) addresses provided by the relevant contributors for airdrop redemption in this discussion.

文档内容贡献者的激励请参考：

https://confluxglobal.notion.site/6665e6288ea942d5aa415c408bcfb145?v=14f867639969811c9551000c92971f4d&pvs=4

其激励根据[2024 reward guidelines](https://github.com/Conflux-Chain/conflux-documentation/blob/2024Q1-reward/docs/general/CONTRIBUTING/reward-guidelines.md)计算。如果您有对应贡献但被遗漏在了名单外，欢迎联系我们。此外我们将在本discussion中收集相关贡献者提供的领取空投的（临时）地址。

Thank you for your assistance in enhancing our project. If you have any queries or require assistance, please feel free to create an issue in our Github repository.

感谢各位持续帮助我们改进项目。如果你还有任何疑问或需要帮助，请随时在我们的Github仓库中创建issue。

**Comment by Vikash-8090-Yadav** [comment]
Hey @darwintree   The payment for the above contribution completed or its still pending ?

**Reply by darwintree** [reply]
Hi, we are checking if anything got stuck, expect to get response before next Monday.

**Reply by darwintree** [reply]
@Vikash-8090-Yadav https://confluxscan.io/transaction/0x617351246b73102929bb3170a01b6a63a7a4d3ac39ab1ff7783845d9a388dbe4

It seems that the reward is already distributed. Would you check it？

**Reply by Vikash-8090-Yadav** [reply]
> @Vikash-8090-Yadav https://confluxscan.io/transaction/0x617351246b73102929bb3170a01b6a63a7a4d3ac39ab1ff7783845d9a388dbe4
> 
> It seems that the reward is already distributed. Would you check it？

Thanks. Recieved.
